lib/solidus_graphql_api.rb

```diff
@@ -9,3 +9,4 @@
 require 'solidus_graphql_api/version'
 require 'solidus_graphql_api/engine'
```

Comment:
This engine is already a Railtie, see https://github.com/rails/rails/blob/master/railties/lib/rails/engine.rb#L348. You can just push the initializer into it.